src/litserve/server.py

```diff
@@ -27,11 +28,16 @@
 from fastapi import FastAPI, Depends, HTTPException, BackgroundTasks, Request, Response
 from fastapi.security import APIKeyHeader
 import sys
+
+from fastapi.responses import StreamingResponse
+
 from litserve import LitAPI
 
 # if defined, it will require clients to auth with X-API-Key in the header
 LIT_SERVER_API_KEY = os.environ.get("LIT_SERVER_API_KEY")
 
+STOP_ITERATION = pickle.dumps(StopIteration())
```

Comment:
converted the `StopIteration()` object into its pickle format to avoid pickling error from `multiprocessing.Pipe`.